azure/citus-bot.sh

```diff
@@ -22,8 +22,6 @@ export RESOURCE_GROUP_NAME="$1"
 
 ./create-cluster.sh
 
-trap - EXIT
-
 ssh_port=$(rg_get_ssh_port ${RESOURCE_GROUP_NAME})
```

Comment:
I think we shouldn't drop that since I feel like we would still want to drop the resource group if something fails around L24-L25, or shouldn't we ?

Reply:
The git diffs are a bit misleading for this one.
On L19 we actually trap the EXIT with the delete resource group script.
This line that I removed was overriding the previous trap with nothing, so when exiting, the resource group was not being cleaned.

Reply:
yeah this is true, so now I'm wondering why we were removing the trap after creating cluster, maybe it's something leftover from a debugging session :)